fe/fe-core/src/main/java/com/starrocks/scheduler/Task.java

```diff
@@ -50,6 +51,15 @@ public class Task implements Writable {
     @SerializedName("source")
     private Constants.TaskSource source = Constants.TaskSource.CTAS;
 
+    // set default to ROOT is for compatibility
```

Comment:
The task framework is new, why need to consider compatibility?

Reply:
Task framework already build in 2.3 version it supported submit statement for async CTAS.